frappe/permissions.py

```diff
@@ -23,7 +23,7 @@ def inner(*args, **kwargs):
 		frappe.flags['has_permission_check_logs'] = []
 		result = func(*args, **kwargs)
 		self_perm_check = True if not kwargs.get('user') else kwargs.get('user') == frappe.session.user
-		raise_exception = False if kwargs.get('raise_exception') == False else True
+		raise_exception = False if kwargs.get('raise_exception') is False else True
```

Comment:
I think this could just be:

```suggestion
		raise_exception = kwargs.get('raise_exception')
```

Reply:
@Alchez This is mostly to set `raise_exception` as True if the value for `raise_exception` is not passed explicitly.

(Am guessing that `raise_exception` will not be available always)

Reply:
Ohh, I see now. I think it might be more readable like this then (slightly longer though):

```suggestion
		raise_exception = kwargs.get('raise_exception') if isinstance(kwargs.get('raise_exception'), bool) else True
```

Reply:
Ohh I see now. I think this could be more readable (though longer) as:

```suggestion
		raise_exception = kwargs.get('raise_exception') if isinstance(kwargs.get('raise_exception'), bool) else True
```

Reply:
I thought about simplifying this part but was afraid to mess up the current logic, so i sticked to the purpose of this PR.

Reply:
That's fair, the expression just seemed confusing at first glance, that's all :sweat_smile: